operator-framework-core/src/main/java/io/javaoperatorsdk/operator/Operator.java

```diff
@@ -188,39 +173,13 @@ public void close() {
       if (configuration == null) {
         configuration = existing;
       }
-
-      final Class<R> resClass = configuration.getCustomResourceClass();
-      final String controllerName = configuration.getName();
-      final var crdName = configuration.getCRDName();
-      final var specVersion = "v1";
-
-      // check that the custom resource is known by the cluster if configured that way
-      final CustomResourceDefinition crd; // todo: check proper CRD spec version based on config
-      if (configurationService.checkCRDAndValidateLocalModel()) {
-        crd = k8sClient.apiextensions().v1().customResourceDefinitions().withName(crdName).get();
-        if (crd == null) {
-          throwMissingCRDException(crdName, specVersion, controllerName);
+      synchronized (lock) {
```

Comment:
Here we still use synchronized keyword as far I can see.

Reply:
Yep, seems like I forgot about that part. Let's remove synchronization for now and re-add it when/if needed.